### PR TITLE
Implement basic ui

### PR DESCRIPTION
**Pull request by TaiwoEnoch**
## This PR is to feature the basic ui of school library, following the steps below ⬇️ 

- [ ] Create a app.rb file that will serve as your console app entry-point. It should have methods that do the following:

- List all books.
- List all people.
- Create a person (teacher or student, not a plain Person).
- Create a book.
- Create a rental.
- List all rentals for a given person id.

- [ ] In your main.rb define the entry point, this will be a method called main that is invoked at the end of your file. This method should do the following:

- Present the user with a list of options to perform.
- Lets users choose an option.
- If needed, ask for parameters for the option.
- Have a way to quit the app.

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -1,17 +1,14 @@
-require_relative './person'
+require './person'
 
 class Student < Person
-  def initialize(age, classroom, name: 'unknown', parent_permission: true)
-    super(age, name: name, parent_permission: parent_permission)
+  attr_accessor :classroom
+
+  def initialize(age, classroom, name = 'Unknow', parent_permission: true)
+    super(name, age, parent_permission: parent_permission)
```
Comment (Munsa1) [inline student.rb]: - [ ] KIndly ensure the app works as expected when creating the student by giving the correct output. Currently your app is showing the following
image

Expected output
image

File: main.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  when '1'
+    app.list_all_books
+  when '2'
+    app.list_all_people
+  when '3'
+    app.create_person
+  when '4'
+    app.create_book
+  when '5'
+    app.create_rental
```
Comment (Munsa1) [inline main.rb]: - [ ] Trying to create a rental results in the following error. The expected arguments and the arguments being given are not matching
image

## Note :red_circle: 
Kindly ensure that the rentals are able to be listed after fixing this error as it is part of the app `menu`  and the requirements

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @TaiwoEnoch, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Able To List Books With The App :heavy_check_mark: 
Able To List People With The App :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._
### Due to insufficient files changed I will leave some comments here :red_circle: 
- [ ] Kindly fix the link to your issues page in the ReadMe file as it points to a different repository
image

- [ ]  It would be nice if your title in the readme file would reflect the current project for this milestone as well
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: student.rb (hunk 1 above)
Comment (KDlamini) [reply student.rb]: Well resolved ✔️ 

image

File: main.rb (hunk 2 above)
Comment (KDlamini) [reply main.rb]: Well resolved ✔️ 

image


Rental can be listed by ID 💯 

image

**Review by KDlamini — APPROVED** [review]
### Hi @TaiwoEnoch 🙋‍♀️,  

Excellent work! 💯 
You have done a great job adding the Basic UI for the OOP school library. 🎊 🥳 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

## Highlights 🌞 

✔️  Good PR title and description
✔️  The `app.rb` and `main.rb` files are set up correctly
✔️  User can create students, teachers, books, and add rentals successfully
✔️  Well documented README

# Status: Approved ✔️ 

oscars-standing-ovation
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**